master/internal/telemetry/telemetry.go

```diff
@@ -27,23 +29,25 @@ type telemetryRPFetcher interface {
 	) (*apiv1.GetResourcePoolsResponse, error)
 }
 
-type telemetryTick struct{}
-
-// TelemetryActor manages gathering and sending telemetry data.
-type TelemetryActor struct {
+// Telemetry manages gathering and sending telemetry data.
+type Telemetry struct {
 	db        db.DB
 	rm        telemetryRPFetcher
 	client    analytics.Client
 	clusterID string
+	syslog    *logrus.Entry
 }
 
+// TelemetryActor is the global telemetry singleton.
+var TelemetryActor *Telemetry
```

Comment:
Since we have an Actor System and this isn't one of those, it seems best to me to avoid using that word to avoid confusion.

Reply:
resolved back to `DefaultTelemetry`